api/events/snapshot.proto

```diff
@@ -26,13 +26,16 @@ option (containerd.plugin.fieldpath_all) = true;
 message SnapshotPrepare {
 	string key = 1;
 	string parent = 2;
+	string snapshotter = 5;
```

Comment:
Did we have 3 and 4 before?

Reply:
The fields only need to be unique and anything in the 1-15 range encodes to the same size. Since it is common across all the events, I choose a number higher than what any use so that it can be common and leave 1-4 reserved for event-specific fields. I am not a protobuf expert so I could be running afoul of some conventions here.